app/assets/stylesheets/application.css

```diff
@@ -1,15 +1,12 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  background-color: #cbc4c4;
+  color: #fff;
+}
+
+nav {
+  background-color: rgba(240, 240, 240);
+  color: #fff;
+}
```

Comment:
- [ ] A very fantastic job 👏 however to be consistent with the project requirements, I think you need to follow the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) by using the same layout, colors, margin, space between items...
a simple video review bellow
https://drive.google.com/file/d/1CmSFAMiZoqOacfljR7BXDqZcgNR4uDO2/view?usp=sharing

Reply:
Thanks for your feedback. I will fix the required changes.